File: service/SurfWeb/Core/Models/PlayerModel.cs
using System.ComponentModel.DataAnnotations;

namespace Core.Models
{
    /// <summary>
    /// 玩家
    /// </summary>
    public class PlayerModel : BaseEntity
    {
        /// <summary>
        /// 腐竹玩家id
        /// </summary>
        public int Auth { get; set; }
        /// <summary>
        /// 玩家名
        /// </summary>
        [MaxLength(128)]
        public required string Name { get; set; }
        /// <summary>
        /// 积分
        /// </summary>
        public decimal Integral { get; set; }
        /// <summary>
        /// 地图完成数
        /// </summary>
        public int SucceesNumber { get; set; }
        /// <summary>
        /// 主线wr数
        /// </summary>
        public int WRNumber { get; set; }
        /// <summary>
        /// 奖励wr数
        /// </summary>
        public int BWRNumber { get; set; }
        /// <summary>
        /// 阶段wr数
        /// </summary>
        public int SWRNumber { get; set; }
    }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/domain/PlayerModel.java
package com.surfweb.system.domain;

import com.surfweb.common.core.domain.BaseEntity;
import com.surfweb.common.enums.RecordTypeEnum;
import lombok.Data;

@Data
public class PlayerModel extends BaseEntity {

    /**
     * 腐竹玩家id
     */
    private int auth;

    /**
     * 玩家名
     */
    private String name;

    /**
     * 积分
     */
    private java.math.BigDecimal integral;

    /**
     * 地图完成数
     */
    private int succeesNumber;

    /**
     * 主线wr数
     */
    private int wrNumber;

    /**
     * 奖励wr数
     */
    private int bwrNumber;

    /**
     * 阶段wr数
     */
    private int swrNumber;
}
